lambda/lambda-returns-usr-info.py

```diff
@@ -0,0 +1,80 @@
+import json
+import boto3
+from botocore.exceptions import ClientError
+import psycopg2
+from psycopg2 import OperationalError
+
+
+def getCredentials():
+    credential = {}
+    
+    secret_name = "mysecretname"
+    region_name = "us-east-1"
+    
+    client = boto3.client(
+      service_name='secretsmanager',
+      region_name=region_name
+    )
+    
+    try:
+        get_secret_value_response = client.get_secret_value(
+            SecretId=secret_name
+        )
+    except ClientError as e:
+        if e.response['Error']['Code'] == 'ResourceNotFoundException':
+            print("The requested failed due to permmisions issue")
+        elif e.response['Error']['Code'] == 'InvalidRequestException':
+            print("The request was invalid due to:", e)
+        elif e.response['Error']['Code'] == 'InvalidParameterException':
+            print("The request had invalid params:", e)
+        elif e.response['Error']['Code'] == 'DecryptionFailure':
+            print("The requested secret can't be decrypted using the provided KMS key:", e)
+        elif e.response['Error']['Code'] == 'InternalServiceError':
+            print("An error occurred on service side:", e)
+
+    else:
+      secret = json.loads(get_secret_value_response['SecretString'])
```

Comment:
I would use `try` `except` block here in case that the secret json is malformed.